marketplace/client.go

```diff
@@ -52,10 +52,12 @@ const SET_APP_TILE = `
 `
 
 const PUBLISH_MODULE = `
-  mutation PublishModule($input: PublishDraftModuleInput!) {
+  mutation PublishModule($input: PublishDraftModuleInputV2!) {
 	publishDraftModule(input: $input) {
```

Comment:
I believe you'll want the V2 resolver here as well

```suggestion
	publishDraftModuleV2(input: $input) {
```

Reply:
Oh shoot, good call. Sorry I had to rewrite this after a git mishap. Thanks!